src/inmanta/ast/entity.py

```diff
@@ -74,7 +74,7 @@ def __init__(self, name: str, namespace: Namespace) -> None:
         self._index = {}  # type: Dict[str,Instance]
         self.index_queue = {}  # type: Dict[str,List[Tuple[ResultVariable, Statement]]]
 
-        self._instance_list = []  # type: List[Instance]
+        self._instance_list = set()  # type: Set[Instance]
```

Comment:
unrelated, but important patch?

Reply:
indeed, seems to have slipped in. was a bugfix for something we looked at